src/serverMain/kotlin/com/studo/campusqr/endpoints/UserFrontend.kt

```diff
@@ -14,17 +12,15 @@ import java.util.*
 
 
 suspend fun ApplicationCall.userFrontend() {
-  val locationId = parameters["l"]
-  val locationName = locationId?.let { id -> getLocationOrNull(id)?.name }
-  val language = this.language
+  val fullLocationId = parameters["l"]
+  var seatNr: Int? = null
+  val locationId = if (fullLocationId?.contains("-") == true) {
+    seatNr = fullLocationId.substringAfterLast("-", "").trimStart('0').toIntOrNull()
```

Comment:
Pls check if we scan a paper with "#001" we also show in the user frontend "#001" and not "#1"

Reply:
ok, github reformatted my message :D

Reply:
> <img width="593" alt="image" src="https://user-images.githubusercontent.com/33554869/93793614-53598d00-fc37-11ea-976e-7e0db52a0173.png">
lol